ginga/util/iqcalc.py

```diff
@@ -501,7 +501,7 @@ def qualsize(self, image, x1=None, y1=None, x2=None, y2=None,
                  edgew=0.01):
 
         x1, y1, x2, y2 = int(x1), int(y1), int(x2), int(y2)
-        data = image.cutout_data(x1, y1, x2, y2, astype='float32')
+        data = image.cutout_data(x1, y1, x2, y2, astype=np.float)
```

Comment:
Is the change from 32-bit to 64-bit float intentional here? Or should it be cast to the native image data type?

Reply:
Say rather that I was struck by the explicit use of `float32`.  I assumed that `np.float` might match more naturally to what the native floating point size is.